scripts/update_golang_version.sh

```diff
@@ -9,16 +9,19 @@ fi
 
 golang_version="$1"
 
-current_version=$(grep -o -e 'golang:[.0-9]*' .circleci/config.yml | head -n1 | cut -d: -f2)
+current_version=$(grep -o -E -e '/golang:[.0-9]+' .circleci/config.yml | head -n1 | cut -d: -f2)
+if [ -z ${current_version} ]; then
```

Comment:
Might need this to appease the linter

```suggestion
if [ -z "${current_version}" ]; then
```

Reply:
Fixed and tested. Thanks!